content/java/concepts/classes/classes.md

```diff
@@ -15,31 +15,34 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A class is a blueprint or a template for an object. Blueprints detail the general structure (information and behavior) about an object.
+A class is a blueprint or template for an object in Java. They detail the general structure and data for an object including information such as properties, attributes, and method behavior.
```

Comment:
```suggestion
 In Java, a class is a blueprint or template for an object. They detail the general structure and data for an object including information such as properties, attributes, and method behavior.
```

Reply:
Either "In Java, classes are a ... They detail..." or "In Java, a class is a ... It details the..."

Reply:
I'm gonna go with the former! 👍